From: Christian Welzel <gawain@camlann.de>
Date: Thu, 08 Nov 2012 22:22:22 +0100
Subject: Set patch level version as debian wheezy version
Description: For better identification of patched debian versions
of TYPO3 add a patch level version to the normal TYPO3 version.
The format "-pl.wheezy{X}" was choosen to make version_compare of
PHP happy.

diff a/t3lib/config_default.php b/t3lib/config_default.php
--- a/t3lib/config_default.php	2012-11-08 19:18:36.263656525 +0100
+++ b/t3lib/config_default.php	2012-11-08 20:52:33.599471678 +0100
@@ -672,7 +672,7 @@
 $T3_VAR = array();	// Initialize.
 
 	// TYPO3 version
-$TYPO_VERSION = '4.5.19';	// deprecated: use the constants defined below
+$TYPO_VERSION = '4.5.19-pl.4.5.34';	// deprecated: use the constants defined below
 define('TYPO3_version', $TYPO_VERSION);
 define('TYPO3_branch', '4.5');
 define('TYPO3_copyright_year', '1998-2012');
